index.html

```diff
@@ -10,46 +10,236 @@
    
-   <a class="LetsConnect" href="#">LET’S CONNECT </a>
+     <ul class="resources" title="resources">
+      <li><a  class="button" href="#">HTML</a> <li>
+      <li><a  class="button" href="#">CSS</a> </li>
+      <li><a  class="button" href="#">javaScript</a></li>      
+    </ul>
+
+    <button onclick="togglePopup3()"  class="normalbutton seeproject">See Project</button>
+    
+    </div>
+  </div>
+
+ <!-- page 5-->
+  <div class="card2" title="works-card-4">
+    <img class="avator4" id="avator" src="sources/Portfolio4.png" alt="Avatar">
+    <div class="container2">
+      <h2><b>Multi-Post</br>
+        Stories</b></h2>
+
+     <ul class="tonic">
+      <li><a class="tonic canopy" href="#">CANOPY</a></li>
+      <li class="tonic dot"></li>
+      <li class="tonic back">Back End Dev</li>
+       <li class="tonic dot"></li>
+       <li class="tonic year">2015</li>
+     </ul>
+
+     <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
    
-   <ul class="topicons">
-     <li><a href="#"> <img src="sources/Vector4.png" alt="twitter"></a></li>    
-     <li><a href="#"> <img src="sources/linkedin.png" alt="linkedin"></a></li>     
-     <li><a href="#"> <img src="sources/Vector5.png" alt="medium"></a></li>        
-     <li><a href="#"> <img src="sources/Vector1.png" alt="github"></a></li>        
-     <li><a href="#"> <img src="sources/Vector3.png" alt="chill"></a></li>   
+     <ul class="resources" title="resources">
+      <li><a  class="button" href="#">HTML</a> <li>
+      <li><a  class="button" href="#">CSS</a> </li>
+      <li><a  class="button" href="#">javaScript</a></li>      
+    </ul>
+
+    <button onclick="togglePopup4()" class="normalbutton2 seeproject">See Project</button> 
+     
+    </div>
+  </div>
+</section>
+
+<!--Page 6-->
+<article class="aboutme" id="about" title="about-me">
+
+<section class="top">
+  <h2 id="heading">About<br>Myself</h2>
+  <p id="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  
+  <div class="socialmedia" title="social-media-accounts">
+   <h4 id="secondheading5">LET’S CONNECT</h4>
+     
+   <ul class="frameicons">
+    <li><a href="#"><img id="twitter"  src="sources/Vector4.png" alt="twitter"></a></li>    
+    <li><a href="#"><img id="linkedin" src="sources/linkedin.png" alt="linkedin"></a></li>     
+    <li><a href="#"><img id="medium"   src="sources/Vector5.png" alt="medium"></a></li>        
+    <li><a href="#"><img id="github"  src="sources/Vector1.png" alt="github"></a></li>        
+    <li><a href="#"><img id="peace"  src="sources/Vector3.png" alt="peace"></a></li>   
    </ul>
+  </div>  
+
+</div>
    
+<div class="action-last" title="get-my-resume-button">
+  <a class="normalbutton3"  href="#">Get my resume</a> 
+</div>
```

Comment:
The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)
Kindly put that in the `button` tag